test/e2e/run-test/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import { Command, flags } from '@oclif/command'
+import chalk from 'chalk'
+import execa from 'execa'
+import Listr from 'listr'
+import emoji from 'node-emoji'
+import { join } from 'path'
+import rp from 'request-promise'
+
+class RunTest extends Command {
+  static description = 'describe the command here'
+
+  static flags = {
+    // add --version flag to show CLI version
+    version: flags.version({ char: 'v' }),
+    help: flags.help({ char: 'h' }),
+    backend: flags.enum({ options: ['1', '2', 'multi'], default: '2' }),
+    local: flags.boolean({ default: false })
+  }
+
+  async run() {
+    const { flags } = this.parse(RunTest)
+
+    this.log(
+      chalk.bold(
+        ` ${emoji.get('robot_face')}  Kuzzle Admin Console End to End tests\n`
+      )
+    )
+
+    switch (flags.backend) {
+      case '1':
+      case '2':
+        await this.singleBackend(flags.backend, flags.local)
+        break
+      case 'multi':
+      default:
+        await this.multiBackend()
+        break
+    }
+  }
+
+  async singleBackend(version: string, local: boolean = false) {
+    this.log(
+      chalk.blueBright(
+        ` Preparing single-backend stack with Kuzzle v${version}`
+      )
+    )
+
+    const tasks = new Listr([
+      {
+        title: `Launch Kuzzle version ${version}`,
+        skip: () => {
+          if (local) {
+            return 'Using local Kuzzle'
+          }
+        },
+        task: () => {
+          const docoFile = join(
+            process.cwd(),
+            'test',
+            'e2e',
+            `stack-v${version}.yml`
+          )
+          const doco = execa('docker-compose', [
+            '-f',
+            docoFile,
+            '-p',
+            `stack-${version}`,
+            'up',
+            '-d'
+          ])
+          // doco.stderr.pipe(process.stderr)
```

Comment:
if useless, remove it

Reply:
I just kept this since I often use it for debugging